ghub.el

```diff
@@ -880,16 +880,12 @@ WARNING: The token will be stored unencrypted in %S.
           ;; simplicity it's better to error out here and ask the user to
           ;; take action. This situation should almost never arise anyway.
           (ghub-http-error
-           (if (string=
-                "already_exists"
-                (let-alist (nth 3 ghub--create-token-error)
-                  (car .errors.code)))
-               (error
-                (concat "A token named %S\nalready exists on Github."
-                        " Please visit "
-                        "https://github.com/settings/tokens and"
-                        " delete it.")
-                ident))))
+           (if (string-equal (let-alist (nth 3 ghub--create-token-error)
```

Comment:
`string=` is the alias and somehow it just looks really weird to me. I can see why others would prefer it over `string-equal` as it is much shorter, but since we are comparing strings, not numbers, `=` seems off to me. And somewhat related commands are also named `string-WORD`, e.g. `string-match` or `string-suffix-p`.